docs/DID Communications Research and Specification/Interactions and Messages.md

```diff
@@ -294,22 +361,392 @@ The <u>authenticator</u> answers with an `authenticationResponse`, providing a `
 
 ```JSON
 authenticationResponse: {
+    "context",
     "thread",
-    "signature"
+    "signature",
+    "callbackURL", // OPTIONAL!
+    "responseRequested", //OPTIONAL!
+    "id", // OPTIONAL!
+    "timing" // OPTIONAL!
 }
 ```
 
 ###### Example(s)
 
 ```JSON
 {
-    "thread": "69-420-1337",
+    "context": "authentication/1.0/authenticationResponse",
+    "thread": "f7771b285a971ba25d66dbe2d82f0bf5f956f4fe548bdf8617c3f24ebc10ed8c",
     "signature": {
         "type": "JcsEd25519Signature2020",
         "verificationMethod": "#authentication",
```

Comment:
Sure, can do - but I don't understand your rationale. The Verifier will already have the Authenticator's DID Document, and can simply look up the key there even without the DID, right? changed in 64593959

Reply:
You can see it as a self. pointer in coding. You can only use self['authentication'] inside the DID Document. Otherwise you have to use the full version. As we are not inside the DID Document, we must use the full DID URL (See [here](https://www.w3.org/TR/did-core/#relative-did-urls)).

Reply:
Will do